.circleci/pyscripts/process_generate.py

```diff
@@ -235,14 +251,26 @@ def add_to_build_list(item):
         What_to_build.clear()
         build_mappings["master-branch"] = True
     elif not build_trigger_override_found and "merge-foundation/" not in branch_name:
-        print("Executing workflow: build-deploy")
-        print()
-        build_mappings["build-deploy"] = mappings["trigger-build"]
+        for item in check_build:
```

Comment:
Can we make this for loop simpler or cleaner ? 
it looks like we have duplicate code which we should be able to clean up.

Reply:
Yes, made some changes.